packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -293,4 +293,5 @@ export default function Autocomplete<
   Multiple extends boolean | undefined = false,
   DisableClearable extends boolean | undefined = false,
   FreeSolo extends boolean | undefined = false,
->(props: AutocompleteProps<T, Multiple, DisableClearable, FreeSolo>): JSX.Element;
+  ChipComponent extends React.ElementType = 'div',
```

Comment:
It should be:
```suggestion
  ChipComponent extends React.ElementType = ChipTypeMap['defaultComponent'],
```

Same in tests

Reply:
addressed